probtorch/stochastic.py

```diff
@@ -342,15 +343,22 @@ def conditioned(self):
                 yield name
 
     def log_joint(self, sample_dims=None, batch_dim=None, nodes=None,
-                  reparameterized=True):
+                  reparameterized=True, sample_dim=None):
         """Returns the log joint probability, optionally for a subset of nodes.
 
         Arguments:
             nodes(iterable, optional): The subset of nodes to sum over. When \
             unspecified, the sum over all nodes is returned.
             sample_dims(tuple): The dimensions that enumerate samples.
+            sample_dim(int): Same with sample_dims, kept for backwards compatibility.
             batch_dim(int): The dimension that enumerates batch items.
         """
+        if (sample_dim is not None):
+            if sample_dims is None:
+                sample_dims = sample_dim
```

Comment:
Shouldn't this contain a single-element tuple as the default when assigning sample_dim to sample_dims?